### PR TITLE
MCSC-248 LTA Protections Pages

### PR DESCRIPTION
**Pull request by ElliotAndersonHMRC**

### CODE REVIEW

File: app/views/lifetimeallowance/LtaProtectionOrEnhancementsView.scala.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@import controllers.lifetimeallowance.routes
+
+@protectionUrl = {
+"https://www.tax.service.gov.uk/gg/sign-in?continue=/protect-your-lifetime-allowance/existing-protections&origin=pensions-lifetime-allowance-frontend"
+}
+
+@contentHint = {
+        <p class = "govuk-body">@messages("ltaProtectionOrEnhancements.hint1")</p>
+        <p><a target="_blank" rel="noopener" href=@protectionUrl> @messages("ltaProtectionOrEnhancements.hint2") </a></p>
```
Comment (bharti5verma) [inline app/views/lifetimeallowance/LtaProtectionOrEnhancementsView.scala.html:34]: Add this class for the vertical line class="govuk-inset-text"

File: app/views/lifetimeallowance/ProtectionReferenceView.scala.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@import controllers.lifetimeallowance.routes
+
+@protectionUrl = {
+"https://www.tax.service.gov.uk/gg/sign-in?continue=/protect-your-lifetime-allowance/existing-protections&origin=pensions-lifetime-allowance-frontend"
+}
+
+@contentHint = {
+<p class = "govuk-body">@messages("protectionReference.hint1")</p>
+<p><a target="_blank" rel="noopener" href=@protectionUrl> @messages("protectionReference.hint2") </a></p>
```
Comment (bharti5verma) [inline app/views/lifetimeallowance/ProtectionReferenceView.scala.html:36]: Add this class for the vertical line class="govuk-inset-text"

File: app/views/lifetimeallowance/ProtectionTypeView.scala.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+@import controllers.lifetimeallowance.routes
+
+@protectionUrl = {
+"https://www.tax.service.gov.uk/gg/sign-in?continue=/protect-your-lifetime-allowance/existing-protections&origin=pensions-lifetime-allowance-frontend"
+}
+
+@contentHint = {
+        <a target="_blank" rel="noopener" href=@protectionUrl> @messages("protectionType.hint") </a></p>
```
Comment (bharti5verma) [inline app/views/lifetimeallowance/ProtectionTypeView.scala.html:33]: Add this class for the vertical line class="govuk-inset-text"

File: app/pages/lifetimeallowance/LtaProtectionOrEnhancementsPage.scala [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.lifetimeallowance
+
+import controllers.{routes => generalRoutes}
+import controllers.lifetimeallowance.{routes => ltaRoutes}
+import models.LtaProtectionOrEnhancements.{Both, Enhancements, Protection}
```
Comment (bharti5verma) [inline app/pages/lifetimeallowance/LtaProtectionOrEnhancementsPage.scala:21]: Unused import

File: test/viewmodels/checkAnswers/lifetimeallowance/ProtectionReferenceSummarySpec.scala [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      )
+    }
+
+    "when answer unavailable, return empty" in {
+      val userAnswers = UserAnswers("id")
+      ProtectionReferenceSummary.row(userAnswers) shouldBe None
+    }
+  }
+
+}
```
Comment (bharti5verma) [inline test/viewmodels/checkAnswers/lifetimeallowance/ProtectionReferenceSummarySpec.scala:60]: Missing SummarySpec file for other 2 screens.

**Review by bharti5verma — CHANGES_REQUESTED** [review]

File: app/views/lifetimeallowance/LtaProtectionOrEnhancementsView.scala.html (hunk 1 above)
Comment (ElliotAndersonHMRC) [reply app/views/lifetimeallowance/LtaProtectionOrEnhancementsView.scala.html]: Done

File: app/views/lifetimeallowance/ProtectionReferenceView.scala.html (hunk 2 above)
Comment (ElliotAndersonHMRC) [reply app/views/lifetimeallowance/ProtectionReferenceView.scala.html]: Done

File: app/views/lifetimeallowance/ProtectionTypeView.scala.html (hunk 3 above)
Comment (ElliotAndersonHMRC) [reply app/views/lifetimeallowance/ProtectionTypeView.scala.html]: Done

File: app/pages/lifetimeallowance/LtaProtectionOrEnhancementsPage.scala (hunk 4 above)
Comment (ElliotAndersonHMRC) [reply app/pages/lifetimeallowance/LtaProtectionOrEnhancementsPage.scala]: Done

File: test/viewmodels/checkAnswers/lifetimeallowance/ProtectionReferenceSummarySpec.scala (hunk 5 above)
Comment (ElliotAndersonHMRC) [reply test/viewmodels/checkAnswers/lifetimeallowance/ProtectionReferenceSummarySpec.scala:60]: Done

**Review by bharti5verma — APPROVED** [review]

**Pull request merged**